pyxem/signals/diffraction2d.py

```diff
@@ -1185,8 +1194,8 @@ def template_match_with_binary_image(
 
         See Also
         --------
-        template_match_disk
-        template_match_ring
+        :meth:`~pyxem.signals.DiffractionSignal2D.template_match_disk`
```

Comment:
I don't think the Sphinx syntax is necessary. Just the full path, like `pyxem.signals.DiffractionSignal2D`.

Reply:
I can't seem to get links like that to work without it...

Reply:
That's weird. Works in kikuchipy: [source](https://github.com/pyxem/kikuchipy/blob/develop/kikuchipy/signals/ebsd.py#L611-L616) -> [docs](https://kikuchipy.org/en/stable/reference/generated/kikuchipy.signals.EBSD.remove_dynamic_background.html).